### PR TITLE
Initial CI dashboard

### PR DESCRIPTION
**Pull request by bobturneruk**
I wanted to bring all the CI statuses into one place. I will need to do some subsequent work to bring the two tables together. I think it's useful enough for everyone to be able to refer to this that it should be merged in it's current, somewhat fragmented and incomplete state.

### CODE REVIEW

File: content/docs/repository_status/_index.md [hunk 1]
```diff
@@ -3,6 +3,20 @@ weight: 7
...
+| javaDataPipeline | [![test with reg](https://github.com/FAIRDataPipeline/javaDataPipeline/actions/workflows/build-test-with-registry.yml/badge.svg)](https://github.com/FAIRDataPipeline/javaDataPipeline/actions/workflows/build-test-with-registry.yml) | [![codecov](https://codecov.io/gh/FAIRDataPipeline/javaDataPipeline/branch/main/graph/badge.svg)](https://codecov.io/gh/FAIRDataPipeline/javaDataPipeline) |
+| pyDataPipeline | [![pyDataPipeline](https://github.com/FAIRDataPipeline/pyDataPipeline/actions/workflows/pyDataPipeline.yaml/badge.svg?branch=dev)](https://github.com/FAIRDataPipeline/pyDataPipeline/actions/workflows/pyDataPipeline.yaml) | [![codecov](https://codecov.io/gh/FAIRDataPipeline/pyDataPipeline/branch/dev/graph/badge.svg)](https://codecov.io/gh/FAIRDataPipeline/pyDataPipeline) |
+| rDataPipeline | [![rDataPipeline](https://github.com/FAIRDataPipeline/rDataPipeline/workflows/build/badge.svg?=1)](https://github.com/FAIRDataPipeline/rDataPipeline/actions) | [![codecov](https://codecov.io/gh/FAIRDataPipeline/rDataPipeline/branch/main/graph/badge.svg?token=xTFk0581AY)](https://codecov.io/gh/FAIRDataPipeline/rDataPipeline) |
```
Comment (RyanJField) [inline content/docs/repository_status/_index.md:18]: The code coverage badges shouldn't need the token.

**Review by RyanJField — CHANGES_REQUESTED** [review]
See comments

File: content/docs/repository_status/_index.md [hunk 2]
```diff
@@ -3,6 +3,20 @@ weight: 7
...
+ 
+
+| Repository | CI Status | Coverage |
+| --- | --- | --- |
+| FAIR-CLI | [![FAIR Data Pipeline CLI](https://github.com/FAIRDataPipeline/FAIR-CLI/actions/workflows/fair-cli.yaml/badge.svg?branch=main)](https://github.com/FAIRDataPipeline/FAIR-CLI/actions/workflows/fair-cli.yaml) | [![codecov](https://codecov.io/gh/FAIRDataPipeline/FAIR-CLI/branch/dev/graph/badge.svg)](https://codecov.io/gh/FAIRDataPipeline/FAIR-CLI) |
+| data-registry | [![FAIR Data Registry](https://github.com/FAIRDataPipeline/data-registry/actions/workflows/fair-data-registry.yaml/badge.svg)](https://github.com/FAIRDataPipeline/data-registry/actions/workflows/fair-data-registry.yaml) | [![codecov](https://codecov.io/gh/FAIRDataPipeline/data-registry/branch/main/graph/badge.svg)](https://codecov.io/gh/FAIRDataPipeline/data-registry) |
+| cppDataPipeline| [![FDP C++ API](https://github.com/FAIRDataPipeline/cppDataPipeline/actions/workflows/fdp_cpp_api.yaml/badge.svg)](https://github.com/FAIRDataPipeline/cppDataPipeline/actions/workflows/fdp_cpp_api.yaml) | [![codecov](https://codecov.io/gh/FAIRDataPipeline/cppDataPipeline/branch/main/graph/badge.svg)](https://codecov.io/gh/FAIRDataPipeline/cppDataPipeline) |
```
[the remark this answers is not in the document]
Comment (bobturneruk) [reply content/docs/repository_status/_index.md:14]: Thanks. I've added this but would eventually like all repos to report coverage in the same way.

File: content/docs/repository_status/_index.md [hunk 3]
```diff
@@ -3,6 +3,20 @@ weight: 7
 title: "Repository Status"
 ---
 
+# CI Dashboard
+
+ 
+
+| Repository | CI Status | Coverage |
+| --- | --- | --- |
+| FAIR-CLI | [![FAIR Data Pipeline CLI](https://github.com/FAIRDataPipeline/FAIR-CLI/actions/workflows/fair-cli.yaml/badge.svg?branch=main)](https://github.com/FAIRDataPipeline/FAIR-CLI/actions/workflows/fair-cli.yaml) | [![codecov](https://codecov.io/gh/FAIRDataPipeline/FAIR-CLI/branch/dev/graph/badge.svg)](https://codecov.io/gh/FAIRDataPipeline/FAIR-CLI) |
```
[the remark this answers is not in the document]
Comment (bobturneruk) [reply content/docs/repository_status/_index.md:12]: I've taken a view for the summary to use the default branch i.e. `develop`. There should be one default deployable branch for each repo. I will bring this up.

File: content/docs/repository_status/_index.md (hunk 1 above)
Comment (bobturneruk) [reply content/docs/repository_status/_index.md:18]: Thanks!

**Comment by bobturneruk** [comment]
Once this is OK, I will propose some changes to the individual repos as it's currently not clear what's passing and what's not due to confusion over what branches to target. Perhaps a standard markdown table at the top of each readme.

**Comment by bobturneruk** [comment]
Sorry @RyanJField - forgot to push so no changes yet!

**Comment by bobturneruk** [comment]
Seems like the java badge needs a token.

**Comment by RyanJField** [comment]
> Seems like the java badge needs a token.

Weird, if it works leave the token in.

**Review by RyanJField — APPROVED** [review]
Sorry forgot to approve

**Comment by bobturneruk** [comment]
Thanks for checking it!

**Pull request merged**